index.tpl.html

```diff
@@ -4,12 +4,9 @@
   <meta charset="UTF-8">
   <title>SpellCasters</title>
   <link rel="icon" type="image/x-icon" href="/img/favicon.ico">
-  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/c3/0.4.11/c3.min.css" />
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <script src="https://cdnjs.cloudflare.com/ajax/libs/howler/2.0.1/howler.core.min.js"></script>
-  <script src='http://d3js.org/d3.v3.min.js'></script>
```

Comment:
Which of these d3/c3 sources are we actually using? If we are just using d3 then we don't need the c3 stylesheet or corresponding script tag.